src/main/java/seedu/address/logic/commands/HistoryCommand.java

```diff
@@ -15,6 +15,8 @@
 public class HistoryCommand extends Command {
 
     public static final String COMMAND_WORD = "history";
+    public static final String COMMAND_ALIAS = "hi";
```

Comment:
you will need to implement the tests for these command alias as well... also take note that it will be good to test for overlapping command alias. this ensures that other developers don't "break" your feature.

Reply:
Since we are using switch-case way to determine the command in AddressBookParser, if 2 command words are the same, the file cannot be compiled. However, will still add tests, and a comment for reminding future developers not to change the switch-case frivolously as it provides a layer of defense against having the same command words, thanks!